easybuild/easyconfigs/r/Rosetta/Rosetta-3.5-ictce-4.1.13.eb

```diff
@@ -13,4 +13,8 @@ sources = ['%(namelower)s%(version)s_bundles.tgz']
 
 builddependencies = [('SCons', '2.3.0', '-Python-2.7.3')]
 
+modextravars = {
+    'ROSETTA3_DB': '$root/rosetta_database',
```

Comment:
This is the line which I used to test it: 
fixbb.mpi.linuxiccrelease -no_binary_dunlib -s 1l2y_centroid.pdb -centroid_input -score:weights centroid_des.wts -out:file:scorefile score.cen_sc
it needs an input file, and the output isn't just a number...